src/Processing/ImportProcessingService.php

```diff
@@ -286,6 +260,51 @@ protected function processElement(string $configName, array $importDataRow, Reso
         }
     }
 
+    /**
+     * Process transformations for an element
+     * 
+     * @param ElementInterface $element
+     * @param array $importDataRow  
+     * @param MappingConfiguration[] $mapping
+     * @return MappingConfiguration|null Returns the mapping that was being processed.
```

Comment:
```suggestion
     * @param MappingConfiguration[] $mapping
```
Thank you for your PR! LGTM, just this part need to remove the reduntant lines and the return was wrong as it's returning void, right?

Reply:
Removed the unneeded lines, and updated the return